changelog/fragments/bump-kubebuilder-add-kustomize.yaml

```diff
@@ -0,0 +1,101 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: Add a plugin (deploy-image/v1-alpha) for Golang projects which scaffolds the code Deploy and Manage an Operand(image). You can test it out by running `operator-sdk --group=example.com --version=v1alpha1 --kind=Memcached --image=memcached:1.6.15-alpine --image-container-command="memcached,-m=64,modern,-v" --image-container-port="11211" --run-as-user="1001" --plugins="deploy-image/v1-alpha"` [More info](https://master.book.kubebuilder.io/plugins/deploy-image-plugin-v1-alpha.html)
```

Comment:
I think we are missing the deprecation go `go/v2` entry in change log. That seems to be important, shouldn't we be adding it? @camilamacedo86

Reply:
That was done in the previous KB release. Could we see to address this one in a follow-up? I will see if we can add in the changelog of the latest SDK release. WDYT?

Reply:
done